planning/behavior_velocity_planner/virtual-traffic-light-design.md

```diff
@@ -0,0 +1,267 @@
+### Virtual Traffic Light
+
+#### Role
+
+Autonomous vehicles have to cooperate with the infrastructures such as:
+
+- Warehouse shutters
+- Traffic lights with V2X support
+- Communication devices at intersections
+- Fleet Management Systems (FMS)
+
+The following items are example cases:
+
+1. Traffic control by traffic lights with V2X support
+   ![traffic_light](docs/virtual_traffic_light/V2X_support_traffic_light.png)
+
+2. Intersection coordination of multiple vehicles by FMS.
+   ![FMS](docs/virtual_traffic_light/intersection_mediation.png)
```

Comment:
Could you update the file name? :pray: 

```suggestion
   ![FMS](docs/virtual_traffic_light/intersection-coordination.png)
```

Also, this can be removed?
![image](https://user-images.githubusercontent.com/31987104/167403699-4984ae8e-792f-4b35-90b6-9e7a3ec0eb6c.png)

Reply:
applied at [e38991e](https://github.com/autowarefoundation/autoware.universe/pull/245/commits/e38991e82ff56f8cd43e95e8f4e6c89c3eadbbf4)